mod-clans/src/main/java/gg/solarmc/loader/clans/ClanLoader.java

```diff
@@ -41,11 +43,12 @@
     @Override
     public OnlineClanDataObject loadData(Transaction transaction, int userId) {
 
-        ClansClanMembershipRecord rec = transaction.getProperty(DSLContext.class).fetchOne(CLANS_CLAN_MEMBERSHIP,CLANS_CLAN_MEMBERSHIP.USER_ID.eq(userId));
+        Optional<Clan> clan = manager.getClanByUser(transaction,userId);
+
+        if (clan.isEmpty()) return new OnlineClanDataObject(userId, manager, null);
```

Comment:
```suggestion
        return new OnlineClanDataObject(userId, manager, clan.orElse(null));
```
Instead of isEmpty/get